src/state/queries/messages/leave-conversation.ts

```diff
@@ -42,7 +36,8 @@ export function useLeaveConvo(
 
       return data
     },
-    onMutate: () => {
+    onMutate: async () => {
+      await queryClient.cancelQueries({queryKey: CONVO_LIST_KEY})
```

Comment:
see example at https://tanstack.com/query/latest/docs/framework/react/guides/optimistic-updates#updating-a-list-of-todos-when-adding-a-new-todo